src/test/groovy/com/onesignal/androidsdk/MainTest.groovy

```diff
@@ -1251,10 +1316,17 @@ class MainTest extends Specification {
         GradleTestTemplate.buildArgumentSets[GRADLE_LATEST_VERSION] = [['build']] // , '--info']]
         when:
         def results = runGradleProject([
+            'android.useAndroidX': true,
+            'android.enableJetifier': true,
             compileLines: """
-                compile 'com.android.support:appcompat-v7:26.0.0'
-                compile 'com.google.firebase:firebase-messaging:15.0.2'
-                compile 'com.google.firebase:firebase-core:16.0.0'
+//                implementation 'com.onesignal:OneSignal:3.13.2'
+                implementation 'com.google.firebase:firebase-messaging:10.2.1'
+
+//                implementation 'com.google.firebase:firebase-iid:17.0.3'
+                implementation 'com.google.firebase:firebase-common:17.1.0'
+
+//            implementation 'com.google.firebase:firebase-messaging:15.0.2'
+//            implementation 'com.google.firebase:firebase-core:16.0.0'
```

Comment:
Why are there a few commented out lines here? Just curious

Reply:
Should `compile 'com.android.support:appcompat-v7:26.0.0'` be flipped to `implementation` as well and commented out?

Reply:
Didn't mean to commit this, I was just testing. Fixed!